magma_access_gateway_installer/agw_installation_errors.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+class AGWInstallationError(Exception):
+    """Base class for Magma AGW installation exceptions."""
+
+    pass
+
+
+class UnsupportedOSError(AGWInstallationError):
+    """Exception raised when installation has been started on an OS other than Ubuntu."""
+
+    def __init__(self):
+        super().__init__("Invalid OS! \n Magma AGW can only be installed on Ubuntu! Exiting...")
+
+
+class InvalidNumberOfInterfacesError(AGWInstallationError):
+    """Exception raised when installation has been started on an OS other than Ubuntu."""
```

Comment:
Has this docstring to be the same as the above? Copy-paste leftover?

Reply:
It is :D Fixed.